Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/ConceptManager.java

```diff
@@ -572,4 +582,11 @@ private int getConceptsCountForConceptList(String listName) {
         alternativeIdService.addAlternativeIds(conceptEntries);
         return conceptEntries;
     }
+
+    @Override
+    public void addWordnetConceptEntry(ConceptEntry entry, String userName)
```

Comment:
why does this need its own method? why not setting the list before adding the concept entry?